{{ cookiecutter.__project_name_kebab }}/.nvmrc

```diff
@@ -1 +1 @@
-{{ cookiecutter.node_version }}
+20
```

Comment:
Can we keep this as a variable and update the variable?

Also, while Node 20 is not the LTS, it almost is so happy for v20. Note that the rest of the Wagtail packages are on 18. 
https://github.com/nodejs/release#release-schedule

Reply:
Thanks.

This got a bit lost as a result of me splitting #51 into smaller chunks to get them finished, but the [original justification](https://github.com/wagtail/cookiecutter-wagtail-package/pull/51#issuecomment-1545191367) for removing this was basically:

Now that we have a `use_frontend` Y/N variable (I've merged https://github.com/wagtail/cookiecutter-wagtail-package/pull/57 and merged main into this branch), this question only really makes sense if you say yes to frontend. Otherwise it is a question that relates to a file that won't be created. Cookiecutter doesn't have the ability to conditionally ask/not ask a question based on a previous answer.

Given it is a variable that is only used in one place, I think on balance it makes sense to just remove it as a variable. Neither solution is perfect though.

Reply:
Fair enough. Happy to keep it simple. Thanks for the investigation.